src/utils/__tests__/mediumAvatar-test.js

```diff
@@ -0,0 +1,14 @@
+import mediumAvatar from '../mediumAvatar';
+
+describe('mediumAvatar', () => {
+  test('If avatarUrl does not contain IMAGE_NAME.png returns avatarUrl as is ', () => {
+    const url = '/avatar/IMAGE_NAME/';
+    expect(mediumAvatar(url)).toEqual(url);
+  });
+
+  test('If avatarUrl contains IMAGE_NAME.png replaces .png with -medium.png in the URL', () => {
```

Comment:
I don't think the test descriptions are clear enough. These are crucial; it is worth taking the time to make them clear.

For example, I am not 100% sure what you mean by the two descriptions here, though I suspect the file extension is the key. Try to explain it more high-level and you don't need to include the exact names and behavior (we should be able to tell by the code itself).